src/pages/Facility/settings/healthcareService/HealthcareServiceForm.tsx

```diff
@@ -463,9 +463,13 @@ function HealthcareServiceFormContent({
               <Button
                 variant="outline"
                 onClick={() =>
-                  navigate(
-                    `/facility/${facilityId}/settings/healthcare_services`,
-                  )
+                  isEditMode && healthcareServiceId
+                    ? navigate(
+                        `/facility/${facilityId}/settings/healthcare_services/${healthcareServiceId}`,
+                      )
+                    : navigate(
+                        `/facility/${facilityId}/settings/healthcare_services`,
+                      )
```

Comment:
```suggestion
 navigate(`/facility/${facilityId}/settings/healthcare_services${isEditMode ? `/${healthcareServiceId}` : ""}`)
```

Isnt this better? dont have to repeat the base path.

Reply:
using suggested code :+1: